src/app/models/AudioEvent.ts

```diff
@@ -33,6 +41,7 @@ export interface IAudioEvent extends HasAllUsers {
   isReference?: boolean;
   taggings?: ITagging[] | Tagging[];
   provenanceId?: Id;
+  errors?: AudioEventError[];
```

Comment:
this is not a property of an audio event. You'll need a different solution here.

I realize the API returns it like this. But yeah, a property for a general model that's only available at one endpoint seems prone to cause further confusion

Reply:
This is a sub-model, so I'll be creating a new submodel on `AudioEventImport` that implements and extends the `AudioEvent` class/interface and extends it with errors